Bio/SeqIO/GckIO.py

```diff
@@ -87,7 +86,7 @@ def _parse(handle):
     if seq_length > length - 4:
         raise ValueError("Conflicting sequence length values")
     sequence = packet[4:].decode("ASCII")
-    record = SeqRecord(Seq(sequence, alphabet=Alphabet.generic_dna))
+    record = SeqRecord(Seq(sequence), annotations={"molecule_type": "DNA"})
```

Comment:
Same here.

Reply:
I'm not 100% sure if GCK is DNA only or could be used with RNA (e.g. virus genomes). So again, point conceded.

Reply:
Done on #3150.